onebusaway-android/src/main/java/org/onebusaway/android/ui/TripModes.java

```diff
@@ -0,0 +1,61 @@
+package org.onebusaway.android.ui;
+
+import org.onebusaway.android.R;
+import org.onebusaway.android.app.Application;
+
+/**
```

Comment:
Could you please add a short description to this class and remove the "Created by...".  Also please add Apache license header.